src/elements/Input/Input.js

```diff
@@ -97,6 +98,10 @@ const Input = ({
 	}
 
 	if(type === 'textarea') {
+		if (checkSensitiveInfo) {
+			/** Checks for SSN and CC#s that 9-19 characters long, ignoring dashes and spaces */
+			other.value = other.value.replace(/(\d[ -]*){9,19}/, '[REDACTED]');
```

Comment:
Instead of overwriting the prop directly, this should be set as it's own variable and then actually add `value` as a prop to `Common`.

so:
1) add `value` as an actual named prop above when defining `Input`.
2) do something like `const sanitizedValue = value.replace(/(\d[ -]*){9,19}/, '[REDACTED]');`
3) add `value={sanitizedValue}` as a named prop (outside of `other`) in `Common` below

And then referencing my above comment about how this needs to work for both textareas and standard inputs, I'd move this outside of these specific checks and put them in the top level of the render function.

Reply:
Made these changes with most recent commit!